data/elan-2a70.tablet

```diff
@@ -0,0 +1,20 @@
+# ELAN touchscreen/pen sensor present in the HP ProBook X360 11 G5 EE 
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/185
+
+[Device]
+Name=ELAN1001
+DeviceMatch=i2c:04f3:2a70
+Class=ISDV4
+Width=10
+Height=6
+IntegratedIn=Display;System
+
+[Features]
+Reversible=true
```

Comment:
```suggestion
```
Doesn't apply here since it's built-in and it's implied anyway (provided you can use the system upside-down :)